src/Stack/Build/ConstructPlan.hs

```diff
@@ -481,7 +481,7 @@ tellExecutables _name (PSFilePath lp)
     | otherwise = return ()
 -- Ignores ghcOptions because they don't matter for enumerating
 -- executables.
-tellExecutables name (PSRemote pkgloc _version _fromSnaphot cp) =
+tellExecutables name (PSRemote pkgloc _version _fromSnapshot cp) =
```

Comment:
I'm hoping this is roughly a variable argument...

Reply:
I think this refactoring is OK.